File: src/test/java/io/mp3editor/util/StringUtilTest.java
package io.mp3editor.util;

import org.testng.annotations.Test;

import static org.testng.Assert.assertEquals;

@Test
public class StringUtilTest {

    private static final String EMPTY_STRING = "";

    @Test
    public void testCleanFileName() {
        String dirtyFileName = "/:\\*\\?\"<>\\|\\\\";
        String cleanFileName = StringUtil.cleanFileName(dirtyFileName);
        assertEquals(cleanFileName, EMPTY_STRING);
    }
}

File: src/test/kotlin/io/mp3editor/util/StringUtilTest.kt
package io.mp3editor.util

import io.mp3editor.util.StringUtil.cleanString
import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test


class StringUtilTest {
    @Test
    fun testCleanFileName() {
        val dirtyFileName = "/:\\*\\?\"<>\\|\\\\"
        val cleanFileName = cleanString(dirtyFileName)
        assertEquals("", cleanFileName)
    }

}
